pkg/pillar/cmd/zedmanager/memorysizemgmt.go

```diff
@@ -40,6 +43,18 @@ func getRemainingMemory(ctxPtr *zedmanagerContext) (uint64, uint64, uint64, erro
 		}
 	}
 	memoryReservedForEve := ctxPtr.globalConfig.GlobalValueInt(types.EveMemoryLimitInBytes)
+	if vault.ReadPersistType() == types.PersistZFS {
+		byteCountStr, err := ioutil.ReadFile("/sys/module/zfs/parameters/zfs_arc_max")
+		if err != nil {
+			log.Errorf("failed to get data from zfs_arc_min. value: %s; code: %v", byteCountStr, err)
```

Comment:
min or max?? No point in printing byteCountStr when the ReadFile fails to read.

Reply:
Corrected